cypress/integration/ete/registration_3.8.cy.ts

```diff
@@ -618,14 +618,15 @@ describe('Registration flow - Split 3/3', () => {
 					timeRequestWasMadeInitialEmail,
 				);
 
-				cy.interceptRecaptcha(3);
+				cy.failRecaptchaRequest();
 				cy.wait(1000); // wait for the send again button to be enabled
 				cy.contains('send again').click();
 				cy.contains('Google reCAPTCHA verification failed.');
 				cy.contains('If the problem persists please try the following:');
 
 				const timeRequestWasMade = new Date();
 				cy.wait(1000); // wait for the send again button to be enabled
+				cy.reEnableRecaptchaRequest();
```

Comment:
Slight refactor of how this test handles recaptca requests .. the test is testing what happens when recpatcha fails and then subsequently works a second time. Instead of intercepting the recpatcha request a set number of times (in this case 3) and returning a 500 response then expecting the 4th request to pass, we now intercept any number of requests and return a 500 up until the point at which we expect the recpatcha to work again where we re-enable the request so that it can pass.